deploy.go

```diff
@@ -194,6 +196,26 @@ func RunDeploy(c *cli.Context) error {
 	return nil
 }
 
+func displayNewCommits(owner string, repo string, c *cli.Context, client *github.Client) {
+	opt := &github.DeploymentsListOptions{
+		Environment: c.String("env"),
+	}
+
+	deployments, _, error := client.Repositories.ListDeployments(owner, repo, opt)
+	if error == nil {
+		sha := *deployments[0].SHA
+		compare, _, cmp_error := client.Repositories.CompareCommits(owner, repo, sha, "master")
```

Comment:
Instead of hard coding master, do we have access to the sha that we're trying to deploy (probably almost always master, but still).